includes/commands/class-mu-migration-import.php

```diff
@@ -650,6 +650,8 @@ private function create_new_site( $meta_data ) {
 		$parsed_url = parse_url( esc_url( $meta_data->url ) );
 		$site_id    = 1;
 
+		$parsed_url['path'] = isset($parsed_url['path'])? $parsed_url['path'] : '/';
```

Comment:
Can you please fix the spacing? This project uses the WordPress coding standards.

```
$parsed_url['path'] = isset( $parsed_url['path'] ) ? $parsed_url['path'] : '/';
```

Reply:
done